multiscene/Readme.md

```diff
@@ -0,0 +1,38 @@
+# Multiscene Example
+
+This example demonstrates the usage of multiscene feature in Wonderland Engine version 1.2.0.
+
+## Overview
+
+Wonderland Engine previously supported managing only a single scene. With the new multiscene feature, you can now easily manage multiple scenes and share resources between them.
+
+## Usage
+
+(in the index.js)
+
+```javascript
```

Comment:
`js`

In the example code, show that `loadSceneGroup` returns the group please:

```js
const sceneGroup = engine.loadSceneGroup('myNewScene.bin');
```